### ISSUE TITLE
Add user to pg_stat_activity

### ISSUE THREAD

**Issue by dusatvoj** [prometheus-community/postgres_exporter#737]
## Proposal
Hello, I'm missing support to count connections to the database per-user. It's nice to see connections per database but when you have multiple users accessing to the database, you can't decide which one is the most consuming one.

You can count by `datname`, like this:
![image](https://user-images.githubusercontent.com/11560794/209794972-26dbd95c-90b8-41d3-a2e7-78319d499e66.png)
But you can't count by `usename`  and `datname`, like this:

```
postgres=# select usename, datname, count(*) from pg_stat_activity group by usename,datname;
      usename      |    datname    | count 
-------------------+---------------+-------
 postgres          | mydb          |     1
 mydb              | mydb          |    15
 postgres_exporter | postgres      |     2
 analytics         | mydb          |     2
 postgres          | postgres      |     1
 postgres_exporter | mydb2         |     1
 mydb              | postgres      |     1
 postfix           | mydb          |     1
 postgres_exporter | mydb          |     1
(9 rows)
```

Yes, I can do it via custom queries, but I think the right approach is via core `pg_stat_activity_.*` metric stuff.

Maybe the easiest way is to add `usename` label to `pg_stat_activity_count` and add `usename` to `group by`.

Thank you

**Comment by TimDurward** [comment]
@dusatvoj  I was looking for this feature as well, looks like it was implemented though, just hasn't been released/built yet.
https://github.com/prometheus-community/postgres_exporter/pull/673

**Comment by dalbani** [comment]
@dusatvoj Did you try with the new released version 0.12.0 of the exporter? It contains the commit/feature in question.
